t/everypolitician_extensions/membership_extension.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+require 'test_helper'
+require_relative '../../lib/everypolitician_extensions.rb'
+
+describe 'MembershipExtensions' do
+  subject do
+    stub_popolo('3df153b', 'Austria/Nationalrat')
+    index_at_known_sha.country('austria')
+                      .legislature('nationalrat')
+                      .term('25')
+                      .legislature
+                      .popolo
+                      .memberships
+                      .first
```

Comment:
I don't think you need so much indirection from legislature to term and back to legislature again there. `country('austria').legislature('nationalrat').popolo.memberships.first` should be enough.